testing/regress/ecl/aggidx1.ecl

```diff
@@ -24,6 +24,8 @@ useSequential := #IFDEFINED(root.useSequential, false);
 
 //--- end of version configuration ---
 
+#option ('obfuscateOutput', true);
```

Comment:
We have test suite that takes so long to run that no-one runs it, because we have a tendency to add a new test just to be sure rather than using coverage analysis to ensure that a particular bit of code is tested. If we take your suggestion to its logical conclusion we would have one test per closed Jira? one test per source code function? 

In general it's good (for working out what has gone wrong) if one test covers one bit of functionality, but we have to balance that against the need to have a test suite that people actually run.